jet-fuel/app/models/url.rb

```diff
@@ -1,7 +1,9 @@
 class Url < ActiveRecord::Base
   before_validation :shorten_url
-  validates :full_url, presence: true, uniqueness: true
-  validates :short_url, presence: true, uniqueness: true
+  validates :full_url,   presence:   true, 
+                       uniqueness:   true,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.